ivy/utils/backend/sub_backend_handler.py

```diff
@@ -200,3 +208,39 @@ def find_available_sub_backends(sub_backends_loc):
             available_sub_backends.append(sub_backend)
 
     return available_sub_backends
+
+
+def available_sub_backend_implementations_for(fn_name: str) -> list:
```

Comment:
I think the `_for` in the function name would be implicitly assumed if we're accepting `fn_name` and the function name is `available_sub_backend_implementations`

Reply:
sure, I'll fix it